### PR TITLE
Fix 'Smartgadget devices notifying temperatures only in Celsius' 

### PR DESCRIPTION
**Pull request by xavier-fernandez**

### CODE REVIEW

**Comment by xavier-fernandez** [comment]
@abrauchli can you take a look to this commit, please?

**Comment by xavier-fernandez** [comment]
Amend done:

bc460b4...0d748f9

**Comment by xavier-fernandez** [comment]
Did an amend 0d748f9...5180d18 

Fixes a typo on the commit message

**Comment by xavier-fernandez** [comment]
Did an amend: 5180d18...69eb21f 

Fixes a merge conflict

File: lib/src/main/java/com/sensirion/libble/services/sensirion/smartgadget/SmartgadgetTemperatureService.java [hunk 1]
```diff
@@ -30,19 +38,72 @@ public SmartgadgetTemperatureService(@NonNull final Peripheral peripheral, @NonN
     }
 
     /**
+     * Checks if the incoming temperature value is feasible considering the device operating ranges
+     *
+     * @param temperatureInCelsius that will be checked
+     * @return <code>true</code> if the temperature value is feasible - <code>false</code> otherwise
+     */
+    private static boolean isTemperatureValueFeasible(@Nullable final Float temperatureInCelsius) {
```
Comment (xavier-fernandez) [inline lib/src/main/java/com/sensirion/libble/services/sensirion/smartgadget/SmartgadgetTemperatureService.java]: @abrauchli 
At the moment this approach do not notify unfeasible temperatures. I can change it to notify that temperatures with a NaN instead.

I can remove the unbox type if it is a problem.

Comment (abrauchli) [inline lib/src/main/java/com/sensirion/libble/services/sensirion/smartgadget/SmartgadgetTemperatureService.java]: ok, let's do it

Comment (xavier-fernandez) [inline lib/src/main/java/com/sensirion/libble/services/sensirion/smartgadget/SmartgadgetTemperatureService.java]: @abrauchli 
Done.

**Comment by xavier-fernandez** [comment]
Amend done: 69eb21f...ceff730

**Comment by abrauchli** [comment]
Merged as 6cb31e4615c09bacae41fa50ac461fd78896f4fa and cbfa3d8d6bbbeb8bb814de7c54bcc72dc361a9c6

**Pull request closed without merging**